tests/parsers/fortran_parser.js

```diff
@@ -0,0 +1,16 @@
+import { expect } from "chai";
```

Comment:
To move to code.js:

- create `utils/parsers` folder
- create `utils/parsers/settings.js` for regex definitions
- move to `utils/parsers/fortran.js` for the function itself